packages/react/menubar/src/Menubar.tsx

```diff
@@ -309,7 +309,7 @@ interface MenubarContentProps extends Omit<MenuContentProps, 'onEntryFocus'> {}
 
 const MenubarContent = React.forwardRef<MenubarContentElement, MenubarContentProps>(
   (props: ScopedProps<MenubarContentProps>, forwardedRef) => {
-    const { __scopeMenubar, ...contentProps } = props;
+    const { __scopeMenubar, align = 'start', ...contentProps } = props;
```

Comment:
I was originally inverting this based on `context.dir` but it turns out that floating already supports this internally.

https://floating-ui.com/docs/computeposition#placement

> The -start and -end alignments are [logical](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Logical_Properties) and will adapt to the writing direction (e.g. RTL) as expected.

Reply:
That's making me double take, cause I did this in `Select` too, and it worked. Is there any harm in doing it ourselves? Thinking that ours responds based on the provider.

Reply:
> Is there any harm in doing it ourselves

If `dir="rtl"` is set on a parent of `Content` then the floating internals will respect that and adjust the alignment automatically, by inverting it ourselves we effectively invert the inversion making resulting in not visual change.

Reply:
That's odd, because I have this in `Select`

https://github.com/radix-ui/primitives/blob/4ae3d7ac55fdc351ad1d23ab3d673b50512e41e2/packages/react/select/src/Select.tsx#L983

and my RTL story looks right…

https://601857eb614bae0021c9b9dd-mipzkchbwo.chromatic.com/?path=/story/components-select--right-to-left

Reply:
> and my RTL story looks right…

> Because it's portalled, apply dir="rtl" to body on that story.

Change addressing this inconsistency in https://github.com/radix-ui/primitives/pull/1890